content/en/docs/contribute/style/page-templates.md

```diff
@@ -0,0 +1,239 @@
+    | whatsnext     | no        |
+
+    The page's body will look like this (remove any optional captures you don't
+    need):
+    
+    ```
+    {% raw %}
+    {{% capture overview %}}
+    
+    {{% /capture %}}
+    
+    {{< toc >}}
+    
+    {{% capture prerequisites %}}
+    
+    {{< include "task-tutorial-prereqs.md" >}} {{< version-check >}}
+    
+    {{% /capture %}}
+    
+    {{% capture steps %}}
+    
+    {{% /capture %}}
+    
+    {{% capture discussion %}}
+    
+    {{% /capture %}}
+    
+    {{% capture whatsnext %}}
+    
+    {{% /capture %}}
+    {% endraw %}
+    ```
+
+- Within each section, write your content. Use the following guidelines:
+  - Use a minimum of H2 headings (with two leading `#` characters). The sections
+    themselves are titled automatically by the template.
+  - For `overview`, use a paragraph to set context for the entire topic.
+  - Add the `{{< toc >}}` shortcode to show an in-page table of contents.
+  - For `prerequisites`, use bullet lists when possible. Add additional
+    prerequisites below the ones included by the `include` in the example
+    above. The default prerequisites include a running Kubernetes cluster.
+  - For `steps`, use numbered lists.
+  - For discussion, use normal content to expand upon the information covered
+    in `steps`.
+  - For `whatsnext`, give a bullet list of up to 5 topics the reader might be
+    interested in reading next.
+
+An example of a published topic that uses the task template is [Using an HTTP proxy to access the Kubernetes API](/docs/tasks/access-kubernetes-api/http-proxy-access-api).
+
+## Tutorial template
+
+A tutorial page shows how to accomplish a goal that is larger than a single
+task. Typically a tutorial page has several sections, each of which has a
+sequence of steps. For example, a tutorial might provide a walkthrough of a
+code sample that illustrates a certain feature of Kubernetes. Tutorials can
+include surface-level explanations, but should link to related concept topics
+for deep explanations.
+
+To write a new tutorial page, create a Markdown file in a subdirectory of the
+`/content/en/docs/tutorials` directory, with the following characteristics:
+
+- In the page's YAML front-matter, set `content_template: templates/tutorial`.
+- In the page's body, set the required `capture` variables and any optional
+  ones you want to include:
+
+    | Variable      | Required? |
+    |===============|===========|
+    | overview      | yes       |
+    | prerequisites | yes       |
+    | objectives    | yes       |
+    | lessoncontent | yes       |
+    | cleanup       | no        |
+    | whatsnext     | no        |
+
+    The page's body will look like this (remove any optional captures you don't
+    need):
+    
+    ```
+    {% raw %}
+    {{% capture overview %}}
+    
+    {{% /capture %}}
+    
+    {{< toc >}}
+    
+    {{% capture prerequisites %}}
+    
+    {{< include "task-tutorial-prereqs.md" >}} {{< version-check >}}
```

Comment:
Why is this basic version check necessary for every task/tutorial? I could understand using a version check if the instructions in the task pertain to a specific version, otherwise, it seems odd to remind the user to check their version without saying why.

Reply:
This was decided by the SIG when the templates were decided upon. Not my new change. :)